File: Data Structures & Algorithms/longest-consecutive-sequence/submission-0.py
class Solution:
    def longestConsecutive(self, nums: List[int]) -> int:
        
        arr = set(nums)
        max_count = 0
        for num in nums:
            if num-1 not in arr:
                count = 0
                for i in range(len(arr)):
                    if num+i not in arr:
                        break
                    count += 1
                
                max_count = max(count, max_count)

        return max_count
            
            

File: Data Structures & Algorithms/duplicate-integer/submission-0.cpp
class Solution {
public:
    bool hasDuplicate(vector<int>& nums) {
        
        std::unordered_set<int> numSet;

        for(auto num : nums)
        {
            if(numSet.count(num))
                return true;
            numSet.insert(num);
        }
        return false;
    }
};